File: src/src/main/java/com/mewebstudio/translatable/dto/request/CategoryCategoryTranslationRequest.java
package com.mewebstudio.translatable.dto.request;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class CategoryCategoryTranslationRequest {
    @NotNull(message = "Name cannot be null")
    @NotBlank(message = "Name cannot be blank")
    private String name;

    private String description;
}

File: src/main/kotlin/com/mewebstudio/translatable/dto/request/CategoryCategoryTranslationRequest.kt
package com.mewebstudio.translatable.dto.request

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull

data class CategoryCategoryTranslationRequest(
    @field:NotNull(message = "Name cannot be null")
    @field:NotBlank(message = "Name cannot be blank")
    val name: String?,

    val description: String?
)
